src/main/java/org/cactoos/iterable/Mapped.java

```diff
@@ -33,6 +33,10 @@
  * @param <X> Type of source item
  * @param <Y> Type of target item
  * @since 0.1
+ * @todo #1169:30m Remove type parameter X from this
+ *  class definition, and put extra type parameter on each ctor.
+ *  Possibly, the same change is possible in iterator.Mapped,
```

Comment:
@andreoss not possibly, this should be done too :) also IMHO there is no reason for those two tasks to be done as once, just say in the todo what is to be done, and then the assignee will decide if everything can be done at once.